src/components/navigation/top-nav/top-nav.tsx

```diff
@@ -47,13 +99,17 @@ const TopNav: React.FC<TopNavProps> = ({ navItems }) => {
 
       {viewportContext.isMobile && (
         <IconButton
-          a11yLabel={'Menu Button'}
+          a11yLabel={`Menu${
+            hasNotifications ? ', criteria has been saved' : ''
+          }`}
           icon={expanded ? Icons.closeOutlined : Icons.menu}
           ariaExpanded={expanded}
           ariaHasPopup={true}
```

Comment:
also, remove aria-haspopup attribute. since this is acting as an accordion, we don't need it/ it's excessive use of aria!